File: class25/java/AP2508-PegadaCarbono.java
public class PegadaCarbono {
    public static void main(String[] x) {
        double kmCarro = 300, kwh = 200;
        double co2 = kmCarro * 0.12 + kwh * 0.09;
        System.out.printf("Pegada estimada: %.1f kg CO2/mes%n", co2);
    }
}

File: class25/python/AP2508-PegadaCarbono.py
# Estimativa simples de pegada de carbono (kg CO2/mes)
km_carro = 300
kwh = 200
co2 = km_carro * 0.12 + kwh * 0.09
print(f"Pegada estimada: {co2:.1f} kg CO2/mes")
